pigar/utils.py

```diff
@@ -81,13 +81,25 @@ def print_table(rows, headers=['PACKAGE', 'CURRENT', 'LATEST']):
 
 
 def parse_reqs(fpath):
-    pkg_v_re = re.compile(r'^(?P<pkg>[^><==]+)[><==]{,2}(?P<version>.*)$')
     """Parse requirements file."""
+    pkg_v_re = re.compile(r'^(?P<pkg>[^><==]+)[><==]{,2}(?P<version>.*)$')
+    referenced_reqs_re = re.compile(r'^-r *(?P<req_f>.*)')
     reqs = dict()
     with open(fpath, 'r') as f:
         for line in f:
             if line.startswith('#'):
                 continue
+            r = referenced_reqs_re.match(line.strip())
+            if r:
+                # Parse referenced requirements file
+                additional_reqs_file = r['req_f']
+                additional_reqs_fpath = os.path.join(
+                    os.path.dirname(fpath), additional_reqs_file)
+                additional_reqs = parse_reqs(additional_reqs_fpath)
```

Comment:
I think to print the filename as context information may be better.

Reply:
@damnever Good idea. I have added this. Right now, the full path is printed to be consistent with the other output.